shell/platform/android/flutter_main.cc

```diff
@@ -155,6 +156,10 @@ void FlutterMain::SetupObservatoryUriCallback(JNIEnv* env) {
       });
 }
 
+static void CreateDefaultFontManager(JNIEnv* env, jclass jcaller) {
+  sk_sp<SkFontMgr> font_mgr(SkFontMgr::RefDefault());
```

Comment:
Remove the `sk_sp<SkFontMgr>` and just call `SkFontMgr::RefDefault()`.  Add a comment explaining that calling `SkFontMgr::RefDefault` will initialize a singleton owned by Skia.

Reply:
done